neo/Cryptography/Helper.cs

```diff
@@ -117,7 +117,8 @@ public static byte[] Sha256(this byte[] value, int offset, int count)
         internal static bool Test(this BloomFilter filter, Transaction tx)
         {
             if (filter.Check(tx.Hash.ToArray())) return true;
-            if (filter.Check(tx.Sender.ToArray())) return true;
```

Comment:
Maybe we can check both, `Sender` and `Witnesses`?

Reply:
`Sender` is one of the `Witnesses`.